azure/cosmos/cosmos_client.py

```diff
@@ -80,21 +80,29 @@ def __init__(self,
         :param documents.ConsistencyLevel consistency_level:
             The default consistency policy for client operations.
 
+        if url_connection and auth are not provided,
+            COSMOS_ENDPOINT and COSMOS_KEY environment variables will be used.
         """
-        self.url_connection = url_connection
 
-        self.master_key = None
-        self.resource_tokens = None
-        if auth != None:
-            self.master_key = auth.get('masterKey')
-            self.resource_tokens = auth.get('resourceTokens')
+        if url_connection and auth:
```

Comment:
what is one of the 2 is present, and the other needs to be fetched from environment?

Reply:
that would be weird that one pass one to the constructor and expect the other one to be read from the env variable. Either both should be passed from env or both through constructor

Reply:
*technically* this breaks customers who were just passing in endpoint before since before `auth` wasn't required to set url_connection. I don't think that's a big deal here.

While I see your point on most folks likely doing both one way or the other, I don't think it makes sense to enforce that and I don't think it is too complicated to support both.

I would prefer just having something like 
```python
self.url_connection = url_conneciton or os.environ["COSMOS_ENDPOINT"]
```
Then handle auth in the else statement of `if auth is not None:`

Reply:
sure, addressed. thanks.